core/base/src/TApplication.cxx

```diff
@@ -628,20 +630,423 @@ TApplication::EExitOnException TApplication::ExitOnException(TApplication::EExit
    return old;
 }
 
-////////////////////////////////////////////////////////////////////////////////
-/// Print help on interpreter.
+/////////////////////////////////////////////////////////////////////////////////
+/// The function generates and executes a command that loads the Doxygen URL in
+/// a browser. It works for Mac, Windows and Linux. In the case of Linux, the
+/// function also checks if the DISPLAY is set. If it isn't, a warning message
+/// and the URL will be displayed on the terminal.
+///
+/// \param[in] url <web page to be displayed in a browser
 
-void TApplication::Help(const char *line)
+void TApplication::OpenInBrowser(const TString &url)
+{
+   // Commands for opening a browser in Linux, Windows and Mac
+   TString cLinux("xdg-open ");
```

Comment:
Why declare them here, and not in their respective `if` block? I.e. you could move this line to line 657, and the others accordingly.